compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -359,6 +361,16 @@ class ROCMTargetBackend final : public TargetBackend {
     pb.registerFunctionAnalyses(fam);
     pb.registerLoopAnalyses(lam);
     pb.crossRegisterProxies(lam, fam, cgam, mam);
+    // Attempt to load pass plugins and register their callbacks with PB.
+    for (auto &PluginFN : passPlugins) {
+      auto PP = llvm::PassPlugin::Load(PluginFN);
+      if (PP) {
+        PP->registerPassBuilderCallbacks(pb);
+      } else {
+        llvm::errs() << "unable to load plugin " << PluginFN << ":"
```

Comment:
use `llvm::report_fatal_error` with a descriptive message (pretend you are a user who may have no idea what this feature is; include what flag may have caused this, etc)

Reply:
Not resolved?